index.js

```diff
@@ -435,6 +509,12 @@ class TuyaDevice extends EventEmitter {
             await this._sendPing();
           }, this._pingPongPeriod * 1000);
 
+          // Automatically ask for dp_refresh so we
+          // can emit a `dp_refresh` event as soon as possible
+          if (this.globalOptions.issueRefreshOnConnect) {
+            this.refresh([4,5,6,18,19,20]);
```

Comment:
this call is out of specs according to js doc and method abiove, or am I blind?

Reply:
And please add a way to self specify a list of known dps if they are known to the application already